runners/spark/src/main/java/org/apache/beam/runners/spark/translation/SparkPCollectionView.java

```diff
@@ -28,7 +28,7 @@
 import org.apache.spark.api.java.JavaSparkContext;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import scala.Tuple2;
+import scala.Tuple3;
```

Comment:
any chance this will grow to Tuple4, ..., in the future? Shall we create a POJO class instead (similar to SideInputBroadcast) and make the value of the Map this POJO class

Reply:
@Abacn
Thank you for the review. I agree that creating a POJO would be better since the Tuple might need to grow in the future.